enabler/src/de/schildbach/pte/NetworkId.java

```diff
@@ -77,5 +77,8 @@
 	SF, SEPTA,
 
 	// Australia
-	SYDNEY, MET
+	SYDNEY, HSL, NZ, SPAIN, BR, BRFLORIPA, MET,
```

Comment:
You added a couple of extra IDs here. Can you edit your PR so that it adds only ONTARIO?